scipy/meson.build

```diff
@@ -11,6 +11,7 @@ inc_np = include_directories(incdir_numpy)
 
 incdir_f2py = incdir_numpy / '..' / '..' / 'f2py' / 'src'
 inc_f2py = include_directories(incdir_f2py)
+nc_f2py = include_directories(incdir_f2py)
```

Comment:
accidental change?

Reply:
Yeah :sweat_smile: 

It should be fixed now.